scripts/thumbnail-generator/generate_thumbnail.py

```diff
@@ -12,7 +12,7 @@ def parse_args():
     parser = argparse.ArgumentParser(description="This script takes a thumbnail image of the given USD file supplied and associates it with the file.")
     parser.add_argument('usd_file', 
```

Comment:
would be nice if all the args were POSIX style, e.g. `--usd-file` instead of `--usd_file`